business_rules/operators.py

```diff
@@ -630,6 +631,17 @@ def is_unique_set(self, other_value):
         self.value[f"result_{uuid4()}"] = results
         return not (False in results)
 
+    @type_operator(FIELD_DATAFRAME)
+    def is_not_unique_within(self, other_value):
```

Comment:
can we change the name to is_unique_relationship and is_not_unique_relationship? Also can you write a positive version of this operator

Reply:
Agree. I have added a new operator.